spec/models/member_authentication_spec.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+describe MemberAuthentication, type: :model do
```

Comment:
in case you weren't aware,`type: model` isn't necessary for spec files inside of `spec/models`